internal/cli/terraform.go

```diff
@@ -82,17 +89,38 @@ func generateTerraformCmdRun(cli *cli, inputs *terraformInputs) func(cmd *cobra.
 			return err
 		}
 
-		if err := generateTerraformConfigFiles(inputs.OutputDIR, data); err != nil {
+		if err := generateTerraformImportConfig(inputs.OutputDIR, data); err != nil {
 			return err
 		}
 
-		cli.renderer.Infof("Terraform config files generated successfully.")
+		if terraformProviderCredentialsAreAvailable() {
+			if err := generateTerraformResourceConfig(cmd.Context(), inputs.OutputDIR); err != nil {
```

Comment:
IMO, this isn't part of the critical path so we can probably continue with the process if this generate step errors. We can simply provide instructions on how to complete the rest of the steps manually.

Reply:
Updated in [`0117602` (#796)](https://github.com/auth0/auth0-cli/pull/796/commits/01176024dfea4bf8c64d41558b234b9e7bb142e6).